bin/update_currencies.php

```diff
@@ -0,0 +1,80 @@
+<?php
+
```

Comment:
Can you please add file headers? I have the feeling they are going to be removed from the project entirely, but should be consistent until then.